### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by tcojean**
This PR is for pushing the release to the master branch. This brings all commits from develop plus the `fix_jacobi` branch and a few commits for adaptation.

DO NOT MERGE THIS. Currently, tests are still ongoing !
https://gitlab.com/ginkgo-project/ginkgo-public-ci/pipelines/89916672

### CODE REVIEW

**Comment by tcojean** [comment]
Here is the generated documentation for the release branch, as a check before actually releasing. Please double check that there are no problems.

https://ginkgo-project.github.io/ginkgo/doc/release/v1.1.0/
https://ginkgo-project.github.io/ginkgo/doc/pdf/release/v1.1.0.pdf

**Comment by hartwiganzt** [comment]
Thanks, Terry!
In the documentation, some tables look awkward, like below, but I feel this should not stop us from releasing...
Screen Shot 2019-10-19 at 2 19 26 PM

**Comment by tcojean** [comment]
Good point, I guess that is because the title for this section in the documentation is set as a `<h4>`, maybe changing to a `<h2>` or `<h3>` would make it better. I will try this out.

**Comment by hartwiganzt** [comment]
@tcojean I think the problem persists - at least in the current version. However, I still approve it as I feel this is acceptable.

**Comment by tcojean** [comment]
I think this is ready. The tools do not show anything terrible. I fixed a few things from clang-tidy and the documentation issue shown by Hartwig. The problem I found was the submenus had different level (black dot vs white dot), but I still kept the submenus and the same layout.

The full pipeline from this PR can be accessed here:
https://gitlab.com/ginkgo-project/ginkgo-public-ci/pipelines/89916672

All the data is available on the dashboard:
https://my.cdash.org/index.php?project=Ginkgo+Project

 Here is the updated documentation:
https://ginkgo-project.github.io/ginkgo/doc/release/v1.1.0/
https://ginkgo-project.github.io/ginkgo/doc/pdf/release/v1.1.0.pdf

The commits which have not been reviewed are the following:
https://github.com/ginkgo-project/ginkgo/pull/368/files/8256a6cc1fd78b1deeabbd92297afcd66d9beaae..7106bb8f0110af3d98dadd556ceb2369f8acf90a

**Comment by yhmtsai** [comment]
@tcojean Does it pick the `fix_jacobi` branch?

**Comment by tcojean** [comment]
@yhmtsai yes this include the commits from the fix jacobi as you can see in the last link

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,29 +1,80 @@
 # Changelog
 
-This file may not always be up to date for the unreleased commits. For a
-comprehensive list, use the following commands:
+This file may not always be up to date in particular for the unrealeased
```
Comment (pratikvn) [inline CHANGELOG.md]: nit: s/unrealeased/unreleased

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,29 +1,80 @@
...
+Supported systems and requirements:
++ For all platforms, cmake 3.9+
++ Linux and MacOS
+  + gcc: 5.3+, 6.3+, 7.3+, 8.1+
+  + clang: 3.9+
+  + Intel compiler: 2017+
+  + Apple LLVM: 8.0+
+  + CUDA module: CUDA 9.0+
++ Windows
+  + MinGW and CygWin: gcc 5.3+, 6.3+, 7.3+, 8.1+
```
Comment (yhmtsai) [inline CHANGELOG.md]: s/CygWin/Cygwin

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,29 +1,80 @@
...
+  + Microsoft Visual Studio: VS 2017 15.7+
+  + CUDA module: CUDA 9.0+, Microsoft Visual Studio
+  + OpenMP module: MinGW or CygWin.
```
Comment (yhmtsai) [inline CHANGELOG.md]: s/CygWin/Cygwin

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,29 +1,80 @@
...
+page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues).
+
+
+### Additions
++ Upper and lower triangular solvers ([#327](https://github.com/ginkgo-project/ginkgo/issues/327), [#336](https://github.com/ginkgo-project/ginkgo/issues/336), [#341](https://github.com/ginkgo-project/ginkgo/issues/341), [#342](https://github.com/ginkgo-project/ginkgo/issues/342)) 
++ New factorization support in Ginkgo, and addition of the ParILU
+  algorithm ([#305](https://github.com/ginkgo-project/ginkgo/issues/305), [#315](https://github.com/ginkgo-project/ginkgo/issues/315), [#319](https://github.com/ginkgo-project/ginkgo/issues/319), [#324](https://github.com/ginkgo-project/ginkgo/issues/324))
++ New ILU preconditioner ([#348](https://github.com/ginkgo-project/ginkgo/issues/348), [#353](https://github.com/ginkgo-project/ginkgo/issues/353))
++ Windows MinGW and Cygwin support ([#347](https://github.com/ginkgo-project/ginkgo/issues/347))
++ Windows Visual studio support ([#351](https://github.com/ginkgo-project/ginkgo/issues/351))
```
Comment (yhmtsai) [inline CHANGELOG.md]: s/studio/Studio

File: README.md [hunk 5]
```diff
@@ -1,10 +1,9 @@
 ![Ginkgo](/assets/logo.png)
 
 [![Build status](https://gitlab.com/ginkgo-project/ginkgo-public-ci/badges/master/build.svg)](https://github.com/ginkgo-project/ginkgo/commits/master)
-[![Coverage](https://codecov.io/gh/ginkgo-project/ginkgo/branch/master/graph/badge.svg)](https://codecov.io/gh/ginkgo-project/ginkgo)
```
[the remark this answers is not in the document]
Comment (tcojean) [reply README.md]: Yes. It is really giving random results whatever the configuration and filters, whereas CDash has the proper settings.

File: README.md [hunk 6]
```diff
@@ -113,11 +132,16 @@ Contributing guidelines can be accessed in our Wiki under the [Developer's
...
 proper commit messages, understanding Ginkgo's library design, relevant C++
-information, and more.
+information, and more. In general, always refer to this page for developer
+information.
+
+### Support
+If you have any question, bug to report or would like to propose a new feature,
+feel free to [create an
+issue on GitHub](https://github.com/ginkgo-project/ginkgo/issues/new). Another possibility
+is to send an email to [Ginkgo's main email address](ginkgo.library@gmail.com)
```
[the remark this answers is not in the document]
Comment (tcojean) [reply README.md]: This is rather internal.

**Comment by tcojean** [comment]
I think most of the typos are fixed.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (hartwiganzt) [reply README.md]: yes, don't add the developer mail here for now.

**Pull request merged**